b3/clients.py

```diff
@@ -1532,6 +1533,20 @@ def getByCID(self, cid):
                 return None
         return None
 
+    def escape_string(value, mapping=None):
+        if isinstance(value, unicode):
+            return _escape_unicode(value)
+        assert isinstance(value, (bytes, bytearray))
```

Comment:
I would not put an assertion in code that is going straight in production. Maybe just return None if the value is not acceptable and handle the case where the escaped string is None.